src/components/Calendar/helpers/__test__/getNextFocusedDate.spec.js

```diff
@@ -0,0 +1,24 @@
+import getNextFocusedDate from '../getNextFocusedDate';
+import getFirstDayMonth from '../getFirstDayMonth';
+
+describe('getNextFocusedDate', () => {
+    it('should return the first day of current month when dates are not set', () => {
+        const result = getFirstDayMonth(new Date());
```

Comment:
when unit test it is best practice to treat the function as an isolated unit and is good to mock external functions calls, so it is better to mock getFirstDayMonth